packages/cli-lib/api/dfs.js

```diff
@@ -0,0 +1,85 @@
+const http = require('../http');
+const fs = require('fs');
+
+const DEVELOPER_FILE_SYSTEM_API_PATH = 'dfs/v1/projects';
+
+/**
+ * Fetch projects
+ *
+ * @async
+ * @returns {Promise}
+ */
+async function fetchProjects(portalId) {
+  return http.get(portalId, {
+    uri: DEVELOPER_FILE_SYSTEM_API_PATH,
+  });
+}
+// TODO: paging?
```

Comment:
Did we want to add paging, or is this a reminder for later?

Reply:
We'll need to add paging but I'll remove the TODO for now